### ISSUE TITLE
Gradient accumulation code

### ISSUE THREAD

**Issue by King4819** [facebookresearch/deit#240]
Can you provide the gradient accumulation code? Since I only have 1 gpu for training, the max batch size that I can use is 256, and the final result can not match  paper's result. So I'm thinking about doing gradient accumulation, thanks!!!
